content/en/docs/concepts/definitions-terms.md

```diff
@@ -0,0 +1,103 @@
+OCI spec reference: [OCI Annotations][oci-annotations]
+
+## OCI Descriptor
+
+A structure describing content, including the media type, a content-addressable digest, the size, and other properties.
+Descriptors are used to describe layers and configuration in a manifest.
+OCI spec reference: [OCI Descriptor][oci-descriptor]
+
+## OCI Index
+
+A higher-level collection of [image manifests][oci-manifest], or other [oci indexes][oci-index], typically used to describe platform-specific (architecture and operating-system) images that can be identified collectively and referred to together.
+The specific image manifests are identified by modified [descriptors][oci-descriptor] with additional properties and restrictions.
+[OCI Artifacts][oci-artifact] are another use-case for indexes where an OCI Index can reference a collection of artifacts, such as an image, a deployment chart, its SBoM and deployment configuration references.
+OCI spec reference: [OCI Image Index][oci-index]
+
+## OCI Manifest
+
+A description of an [artifact][oci-artifact].
+The manifest references optional configuration and blobs (layers) as content-addressable references ([descriptors][oci-descriptor]).
+Artifacts may be container images, helm charts or other artifact types that may be signed and stored in a registry.
+OCI spec reference: [OCI Image Manifest][oci-manifest]
+
+## Registry
+
+A registry is a collection of [OCI Indexes][oci-index] and [OCI Manifests][oci-manifest] for a specific org or entity.
+A unique registry may be uniquely identified with a domain or a root namespace:  
+![registry name](/docs/unique-registry-domain-ref.png)
+![registry and namespace](/docs/unique-registry-namespace-ref.png)
+
+## Repo/Repository
+
+A repo/repository refers to the unique location, within a registry.
+When referring to a repo, the registry is assumed, based on the context.
+![registry and repository](/docs/registry-repo-ref.png)
+
+## Repository Path
+
+A path, within the unique registry org, up to, but not including the repository (repo) name.
+![repository path](/docs/registry-namespace-ref.png)
+
+## Registry: Fully Qualified Reference
+
+In order to deploy an artifact, a fully qualified reference is required.
+This includes the unique registry, namespace, repo and tag.
+![fully qualified reference](/docs/fully-qualified-artifact-ref.png)
+
+## SBoM
+
+Represents a generic, non-project specific, reference to a Software Bill of Materials.
+Similar to the automotive industry which tracks the components that make up a vehicle, the SBoM may contain the list of packages used within the artifact, the compiler and version used to build the artifact and other relevant information.
+Additional reference: [Software bill of materials][sbom]
+
+## Subject
+
+The data that is signed.
+
+## Tag
+
+Information that represents a version, or platform specific version of the artifact.
+E.g.:
+
```

Comment:
It would be better to add two terms for Trust Store and Trust Policy here. These two terms were defined by Notary maintainers but will be used in the user manual frequently.

Reply:
fixed